packages/venia-concept/src/components/App/renderRoutes.js

```diff
@@ -11,7 +11,11 @@ const renderRoutes = () => (
     <Switch>
         <Route exact path="/search.html" component={Search} />
         <Route exact path="/create-account" component={CreateAccountPage} />
-        <Route render={() => <Page>{renderRoutingError}</Page>} />
+        <Route
+            render={() => (
+                <Page using={{ Route, Router }}>{renderRoutingError}</Page>
```

Comment:
More of a curiosity question: is it possible for the `Route` component being rendered here and the `Route` being passed in the `using` prop to `Page` to be different? Would you ever actually want to do that?

Reply:
It's _possible_. And honestly it's easier to just drill the props through. `<Page />` is its own component and could be used separately by something in the future, so rather than try to make a new ContextProvider and think about all those use cases, I thought props would be simpler code.